packages/plugin-ext/src/main/browser/dialogs/style/modal-notification.css

```diff
@@ -26,10 +26,10 @@
     clear: both;
     box-sizing: border-box;
     position: relative;
-    width: 100%;
     min-width: 200px;
-    background-color: var(--theia-editorWidget-background);
+    max-width: min(66vw, 800px);
     min-height: 35px;
+    background-color: var(--theia-editorWidget-background);
```

Comment:
This is an unnecessary change. Please restore to the [original](https://github.com/eclipse-theia/theia/pull/8080/files#diff-3ff0cf7e13a83017520974a95117399eL31).